### PR TITLE
Update ITEM.tsv [Lines 005530 - 005541]

### PR DESCRIPTION
**Pull request by WorksWellWithOthers**
Make changes to ITEM.tsv.
From Line 20150717_005530 to 20150717_005541

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -5527,18 +5527,18 @@ ITEM_20150717_005526	- 2% chance to deal 600% increased damage
...
-ITEM_20150717_005533	
-ITEM_20150717_005534	
-ITEM_20150717_005535	
-ITEM_20150717_005536	
-ITEM_20150717_005537	
-ITEM_20150717_005538	
-ITEM_20150717_005539	
-ITEM_20150717_005540	
-ITEM_20150717_005541	
+xITEM_20150717_005530   The strength of the Deathweaver is nestled within.
```
Comment (Soukyuu) [inline ITEM.tsv]: Looks like an unintentional space at line start.

**Comment by Soukyuu** [comment]
We don't use x at line start anymore. See #900

**Comment by WorksWellWithOthers** [comment]
Should I remove the x at line start then resubmit a pull request?

**Comment by terrapin47** [comment]
If you push to your master, this PR will reflect those same changes.

**Comment by WorksWellWithOthers** [comment]
Thank you.

**Comment by Soukyuu** [comment]
Btw, you shouldn't be working on master, else you will only be able to have one PR at a time. You should leave master untouched and only bring it up-to-date with ToS:master once something changes. Create branches for each of your PRs instead. (Discord has some git related tutorials linked in #announcements)

**Comment by WorksWellWithOthers** [comment]
Thank you. I will branch away from master.

**Comment by terrapin47** [comment]
Here is a helpful link to keep the master of your fork up-to-date with the original repo's master.
https://help.github.com/articles/syncing-a-fork/

Whenever a pull request is accepted, it would be a good idea to update your master.  This typically resolves the problem where your pull request ends up having merge conflicts, and can't be accepted.

The easiest way for you to tell is to look at your own fork of the repo on the master branch, and check if it says that it is any commits behind Treeofsavior:master.

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**